transforms/pair.go

```diff
@@ -0,0 +1,137 @@
+package transforms
+
+import (
+	"github.com/facebookgo/errgroup"
+	"gopkg.in/Clever/optimus.v3"
+)
+
+// PairType is the type of join to use when Pairing
+type PairType int
+
+// RowHasher takes in a row and returns a hash for that Row.
```

Comment:
I think if I saw a function called `RowAttribute` I wouldn't know what it was for, but if I saw something called `RowHasher` I would... Isn't the identity function considered one type of hashing?